src/www/widgets/api/plugins/temperature.inc

```diff
@@ -32,7 +32,8 @@
 function temperature_api()
 {
     $result = array();
-    exec("/sbin/sysctl -a | grep temperature", $sysctlOutput);
+    exec("sysctl sysctl dev.cpu hw.acpi.thermal | egrep -E 'cpu\.[0-9]+\.temp|tz[0-9]+\.temp'",$sysctlOutput);
```

Comment:
This should use grep, not egrep.

Reply:
Why?

Reply:
Apart from GNU comment? `egrep` is the equivalent of `grep -E`. `egrep -E` is redundant. I'd just go with `grep`. Suppose it doesn't really matter.